### ISSUE TITLE
pro controller not connecting

### ISSUE THREAD

**Issue by flamersy** [Davidobot/BetterJoy#477]
so my pro controller does not want to connect but for some reason the joycons can please help i tried everything

**Comment by szszoke** [comment]
I am also having issues with `v6.3`. I am pretty sure that it is paired correctly because it works with `v5.0`.

Is there a way to generate logs that might be useful for debugging?

**Comment by szszoke** [comment]
After playing around in Windows and the settings of BetterJoy I was able to make my controller to work.

First thing that made it work was to go to the Device Manager and uninstall the `Pro Controller` Bluetooth device. Then I launched Steam without BetterJoy running. Steam detected my Pro Controller but after I quit Steam, BetterJoy couldn't see my controller again. Nor Steam.

Another thing I tried which made the controller work again was to check `UseHIDG` in the settings. If you don't know how to open the settings in BetterJoy, press `>` on the right side of the window around the middle.

Unchecking `UseHIDG` didn't break the connection so I'm thinking that there is some weirdness going on.

It might be some conflict between BetterJoy and Steam. It would be nice to get this clarified by the owner of this project.
